motr/client.h

```diff
@@ -630,15 +641,10 @@ enum m0_entity_type {
 	 *   m0_entity_open(NULL, &obj->ob_entity, &ops[0]);
 	 */
 	M0_ENF_META = 1 << 0,
-	/**
-	 * If this flags is set during entity_create() that means application
-	 * do not support update operation. This flag is not in use yet.
-	 */
-	M0_ENF_NO_RMW =  1 << 1,
```

Comment:
Keep this flag, this is being set by S3,
```  start = data.c_str();
  for (p = start; left > 0; left -= bs, p += bs, offset += bs) {
    if (left < bs) {
      bs = this->get_optimal_bs(left);
      mobj->ob_entity.en_flags |= M0_ENF_NO_RMW;
    }```
    https://github.com/Seagate/cortx-rgw/blob/main/src/rgw/rgw_sal_motr.cc
    
    
    And set flag M0_OOF_NO_RMW, if M0_ENF_RMW is set.

Reply:
@yeshpal-jain-seagate mentioned to add flags in a way so that we can trigger no RMW per io launch, so I have moved this flag from m0_entity_flags to m0_op_obj_flags. Are you saying that we should have both flags?

Reply:
Yes.